core/src/main/java/com/valtech/aemsaas/core/models/responses/search/ResponseHeader.java

```diff
@@ -0,0 +1,16 @@
+package com.valtech.aemsaas.core.models.responses.search;
+
+import com.google.gson.annotations.SerializedName;
+import java.util.Map;
+import lombok.Value;
+
+@Value
+public class ResponseHeader {
+
+  int status;
+
+  @SerializedName("QTime")
+  int qTime;
```

Comment:
what is qTime?

Reply:
I assume query execution time. It is a property contained in the json response.